package.json

```diff
@@ -27,10 +27,10 @@
   },
   "peerDependencies": {
     "chai": "^3.0.0 || ^4.0.0",
-    "cheerio": "0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.1",
-    "enzyme": "1.x || ^2.3.0",
-    "react": "^0.14.0 || ^15.0.0-0",
-    "react-dom": "^0.14.0 || ^15.0.0-0"
+    "cheerio": "0.19.x || 0.20.x || 0.22.x || 1.0.0-rc.2",
```

Comment:
enzyme 3 requires cheerio v1 or higher - this needs to change to `^1.0.0-rc.2`

Reply:
I think #196 has got the dependencies pretty much right

Reply:
yes as a nonbreaking change #196 does this much better (altho enzyme 2 requires cheerio 0.22 so the 0.19 and 0.20 peer deps aren't helpful)